test/protocol/TwinHandlerTest.js

```diff
@@ -377,12 +377,10 @@ describe("IBosonTwinHandler", function () {
       context("💔 Revert Reasons", async function () {
         it("The twins region of protocol is paused", async function () {
           // Pause the twins region of the protocol
-          await pauseHandler
-            .connect(pauser)
-            .pause([PausableRegion.Offers, PausableRegion.Twins, PausableRegion.Bundles]);
+          await pauseHandler.connect(pauser).pause([PausableRegion.Twins]);
 
           // Attempt to Remove a twin, expecting revert
-          await expect(twinHandler.connect(assistant).removeTwin(twin.id)).to.revertedWith(RevertReasons.REGION_PAUSED);
+          await expect(twinHandler.connect(assistant).createTwin(twin)).to.revertedWith(RevertReasons.REGION_PAUSED);
```

Comment:
Why did you remove the `removeTwin` test?

Reply:
This line was wrong since it's inside `createTwin()` context. There were two tests for paused regions that used removeTwin and none that used `createTwin`.

Test for pausable region for `removeTwin` is in line 724.